(DOCS-4112) Add NetFlow beta docs (#15617)
DataDog/documentation: content/en/network_monitoring/devices/netflow.md

---
title: NetFlow Monitoring

> kind: documentation

Reviewer: ```suggestion
kind: documentation
is_beta: true
```
We've been adding is_beta: true to our other docs. It no longer stops the page from being indexed. I think there might be some other functionality linked to it, like an icon or something, that the web team is going to fix for us.

further_reading:
- link: "/network_monitoring/devices/profiles"
  tag: "Documentation"
  text: "Using Profiles with Network Device Monitoring"
---

> {{< beta-callout url="#" btn-hidden="true" d-toggle="modal" d_target="#signupModal" custom_class="sign-up-trigger">}}

Reviewer: ```suggestion
{{< beta-callout url="#" btn_hidden="true" d-toggle="modal" d_target="#signupModal" custom_class="sign-up-trigger">}}
```

NetFlow Monitoring for Datadog Network Device Monitoring is in private beta.
{{< /beta-callout >}}

## Overview

Use NetFlow Monitoring in Datadog to visualize and monitor your flow records from your Netflow-enabled devices.

## Installation

To use NetFlow Monitoring with Network Device Monitoring, ensure you are using the [Agent][1] version 7.40 or newer.

**Note:** Configuring [metric collection from Network Device Monitoring][2] is not a requirement for sending NetFlow data, although it is recommended.

## Configuration

> To configure your devices to send NetFlow, sFlow, or IPFIX traffic to Agent NetFlow Server, your devices must be configured to send traffic to the IP address that the Datadog Agent is installed on, specifically the `flow_type` and `port`.

Reviewer: ```suggestion
To configure your devices to send NetFlow, sFlow, or IPFIX traffic to Agent NetFlow Server, your devices must be configured to send traffic to the IP address that the Datadog Agent is installed on, specifically the `flow_type` and `port`.
```
Should this be "the/an" Agent NetFlow Server? Or is there only one thing called Agent NetFlow Server where all the traffic goes?

---
title: NetFlow Monitoring
kind: documentation
is_beta: true
further_reading:
- link: "/network_monitoring/devices/profiles"
  tag: "Documentation"
  text: "Using Profiles with Network Device Monitoring"
---

> {{< beta-callout url="#" btn_hidden="true" d-toggle="modal" d_target="#signupModal" custom_class="sign-up-trigger">}}
> NetFlow Monitoring for Datadog Network Device Monitoring is in private beta.
> {{< /beta-callout >}}

Author: ```suggestion
<div class="alert alert-warning">NetFlow Monitoring for Datadog Network Device Monitoring is in private beta.</div>
```

## Overview

Use NetFlow Monitoring in Datadog to visualize and monitor your flow records from your Netflow-enabled devices.

## Installation

To use NetFlow Monitoring with Network Device Monitoring, ensure you are using the [Agent][1] version 7.40 or newer.

**Note:** Configuring [metric collection from Network Device Monitoring][2] is not a requirement for sending NetFlow data, although it is recommended.

## Configuration

> To configure your devices to send NetFlow, sFlow, or IPFIX traffic to the Agent NetFlow Server, your devices must be configured to send traffic to the IP address that the Datadog Agent is installed on, specifically the `flow_type` and `port`.

Author: ```suggestion
To configure your devices to send NetFlow, sFlow, or IPFIX traffic to the Agent NetFlow server, your devices must be configured to send traffic to the IP address that the Datadog Agent is installed on, specifically the `flow_type` and `port`.
```
